Make Verb requirements more explicit (#653)
AICC/CMI-5_Spec_Current: cmi5_spec.md

* The "Terminated" verb MUST be the last statement (cmi5 allowed or defined).

AU Verb Ordering Rules within a Registration (per AU) are as follows:

> * More than one "Completed" cmi5 defined statement MUST NOT be used per registration.

Reviewer: ```suggestion
* Exactly zero or one "Completed" cmi5 defined statement MUST be used per registration.
```

> * More than one "Passed" cmi5 defined statement MUST NOT be used per registration.

Reviewer: ```suggestion
* Exactly zero or one "Passed" cmi5 defined statement MUST be used per registration.
```
